roles/content_view_version_cleanup/README.md

```diff
@@ -0,0 +1,45 @@
+theforeman.foreman.content_view_version_cleanup
+===============================================
+
+Clean up unused Content View Versions.
+
+Requirements
+------------
+
+- `theforeman.forman` collection
+- `jmespath` Python library
```

Comment:
Is this common on systems? Or possible to avoid an additional requirement on users? We just got through vendoring our only dependency :/

Reply:
We still depend on requests, we just got rid of a dependency that's not in the base os.

Jmespath is a soft dependency of Ansible, so I'd say it's common.

Reply:
Well we still depend on requests. But that's supposed to be common.

Reply:
Is Jmespath common? In base OS for EL7 and El8 for example?

@evgeni can we make it a requirement of the RPM based install?

Reply:
I think it's in the Ansible repo, our satellite connector role also depends on it.

Reply:
dropped that req